src/sentry/api/endpoints/organization_events_root_cause_analysis.py

```diff
@@ -55,6 +57,7 @@ def query_spans(transaction, regression_breakpoint, params):
     )
     builder.columns.append(Function("countDistinct", [Column("event_id")], "transaction_count"))
     builder.groupby.append(Column("period"))
+    builder.limitby = LimitBy([Column("period")], limit // 2)
```

Comment:
I think you can pass this directly to the `QueryBuilder`

Reply:
`limitby=["period", limit // 2]` gives me `sentry.utils.snuba.SnubaError: Parsing error on line 1 at 'gs[period] LI'` and other ways of passing a `LimitBy` instance into this field don't seem to work so I'm going to keep it like this for now :pray: